Documentation/dev/developer-guide.md

```diff
@@ -111,30 +102,33 @@ make integration REPORTING_OPERATOR_IMAGE_TAG=pr-1234 METERING_OPERATOR_IMAGE_TA
 
 ## Go Dependencies
 
-We use [dep](https://golang.github.io/dep/docs/introduction.html) for managing
+We use `go mod` for managing
 dependencies.
 
-Dep installs dependencies into the `vendor/` directory at the
+`go mod` installs dependencies into the `vendor/` directory at the
 root of the repository, and to ensure everyone is using the same dependencies,
 and ensure that if dependencies disappear, we commit the contents of `vendor/`
 into git.
 
 ### Adding new dependencies
 
-To add a new dependencies, you can generally follow the dep documentation.
-Start by reading [https://golang.github.io/dep/docs/daily-dep.html](https://golang.github.io/dep/docs/daily-dep.html)
-and you it should cover the most common things you'll be using dep for.
+To add a new dependencies, you can generally do the following:
+```
```

Comment:
```suggestion
```bash
```

Reply:
It looks like we're pretty inconsistent with adding syntax highlighting to code blocks, so we can either update all of the tag all of the codeblocks in this doc, or just ignore this suggestion.

Reply:
updated all in this doc